.travis.yml

```diff
@@ -9,20 +9,35 @@ go:
 matrix:
   allow_failures:
     - go: master
+    - env: MONGODB_IMAGE=perconalab/percona-server-mongodb:3.6
```

Comment:
@EvgeniyPatlan I'm having troubles with this image:
```terminal
$ travis show 216
Build #216:  Fix order.
State:         failed
Type:          pull request
Branch:        master (PR #92)
Compare URL:   https://github.com/percona/mongodb_exporter/pull/92
Duration:      1 hrs 12 sec
Started:       2018-03-01 23:37:53
Finished:      2018-03-01 23:55:44

#216.1 passed:   3 min 2 sec    env: MONGODB_IMAGE=mongo:3.0, os: linux
#216.2 passed:   4 min 37 sec   env: MONGODB_IMAGE=mongo:3.2, os: linux
#216.3 passed:   2 min 27 sec   env: MONGODB_IMAGE=mongo:3.4, os: linux
#216.4 passed:   2 min 16 sec   env: MONGODB_IMAGE=mongo:3.6, os: linux
#216.5 passed:   2 min 9 sec    env: MONGODB_IMAGE=percona/percona-server-mongodb:3.0, os: linux
#216.6 passed:   2 min 20 sec   env: MONGODB_IMAGE=percona/percona-server-mongodb:3.2, os: linux
#216.7 passed:   2 min 24 sec   env: MONGODB_IMAGE=percona/percona-server-mongodb:3.4, os: linux
#216.8 failed:   3 min 23 sec   env: MONGODB_IMAGE=perconalab/percona-server-mongodb:3.6, os: linux
#216.9 passed:   4 min 51 sec   env: MONGODB_IMAGE=mongo:3.0, os: linux
#216.10 passed:  6 min 8 sec    env: MONGODB_IMAGE=mongo:3.2, os: linux
#216.11 passed:  4 min 46 sec   env: MONGODB_IMAGE=mongo:3.4, os: linux
#216.12 passed:  4 min 32 sec   env: MONGODB_IMAGE=mongo:3.6, os: linux
#216.13 passed:  4 min 8 sec    env: MONGODB_IMAGE=percona/percona-server-mongodb:3.0, os: linux
#216.14 passed:  4 min 52 sec   env: MONGODB_IMAGE=percona/percona-server-mongodb:3.2, os: linux
#216.15 passed:  4 min 2 sec    env: MONGODB_IMAGE=percona/percona-server-mongodb:3.4, os: linux
#216.16 failed:  4 min 15 sec   env: MONGODB_IMAGE=perconalab/percona-server-mongodb:3.6, os: linux
```

Logs for `perconalab/percona-server-mongodb:3.6`:
```
$ docker-compose logs
Attaching to mongodbexporter_mongo_1
mongo_1  | 2018-03-01T23:06:40.372+0000 I ACCESS   [main] Initialized External Auth Session
mongo_1  | 2018-03-01T23:06:40.392+0000 I CONTROL  [initandlisten] MongoDB starting : pid=1 port=27017 dbpath=/data/db 64-bit host=615f3b28102c
mongo_1  | 2018-03-01T23:06:40.392+0000 I CONTROL  [initandlisten] db version v3.6.0-1.0
mongo_1  | 2018-03-01T23:06:40.392+0000 I CONTROL  [initandlisten] git version: 9d8e32416b9af887bda7fabc98eeffc09109df5b
mongo_1  | 2018-03-01T23:06:40.392+0000 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.1t  3 May 2016
mongo_1  | 2018-03-01T23:06:40.392+0000 I CONTROL  [initandlisten] allocator: tcmalloc
mongo_1  | 2018-03-01T23:06:40.392+0000 I CONTROL  [initandlisten] modules: none
mongo_1  | 2018-03-01T23:06:40.392+0000 I CONTROL  [initandlisten] build environment:
mongo_1  | 2018-03-01T23:06:40.392+0000 I CONTROL  [initandlisten]     distarch: x86_64
mongo_1  | 2018-03-01T23:06:40.392+0000 I CONTROL  [initandlisten]     target_arch: x86_64
mongo_1  | 2018-03-01T23:06:40.392+0000 I CONTROL  [initandlisten] options: {}
mongo_1  | 2018-03-01T23:06:40.393+0000 I STORAGE  [initandlisten] 
mongo_1  | 2018-03-01T23:06:40.393+0000 I STORAGE  [initandlisten] ** WARNING: Using the XFS filesystem is strongly recommended with the WiredTiger storage engine
mongo_1  | 2018-03-01T23:06:40.393+0000 I STORAGE  [initandlisten] **          See http://dochub.mongodb.org/core/prodnotes-filesystem
mongo_1  | 2018-03-01T23:06:40.393+0000 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=1461M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
mongo_1  | 2018-03-01T23:06:40.478+0000 I CONTROL  [initandlisten] 
mongo_1  | 2018-03-01T23:06:40.478+0000 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
mongo_1  | 2018-03-01T23:06:40.478+0000 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
mongo_1  | 2018-03-01T23:06:40.478+0000 I CONTROL  [initandlisten] **          You can use percona-server-mongodb-enable-auth.sh to fix it.
mongo_1  | 2018-03-01T23:06:40.479+0000 I CONTROL  [initandlisten] 
mongo_1  | 2018-03-01T23:06:40.479+0000 I CONTROL  [initandlisten] ** WARNING: This server is bound to localhost.
mongo_1  | 2018-03-01T23:06:40.479+0000 I CONTROL  [initandlisten] **          Remote systems will be unable to connect to this server. 
mongo_1  | 2018-03-01T23:06:40.479+0000 I CONTROL  [initandlisten] **          Start the server with --bind_ip <address> to specify which IP 
mongo_1  | 2018-03-01T23:06:40.479+0000 I CONTROL  [initandlisten] **          addresses it should serve responses from, or with --bind_ip_all to
mongo_1  | 2018-03-01T23:06:40.479+0000 I CONTROL  [initandlisten] **          bind to all interfaces. If this behavior is desired, start the
mongo_1  | 2018-03-01T23:06:40.479+0000 I CONTROL  [initandlisten] **          server with --bind_ip 127.0.0.1 to disable this warning.
mongo_1  | 2018-03-01T23:06:40.480+0000 I CONTROL  [initandlisten] 
mongo_1  | 2018-03-01T23:06:40.480+0000 I STORAGE  [initandlisten] createCollection: admin.system.version with provided UUID: b64c2db3-f372-47a0-a6ba-1fb64146d7f9
mongo_1  | 2018-03-01T23:06:40.496+0000 I COMMAND  [initandlisten] setting featureCompatibilityVersion to 3.6
mongo_1  | 2018-03-01T23:06:40.500+0000 I STORAGE  [initandlisten] createCollection: local.startup_log with generated UUID: bcd77220-d1a3-4cc5-8366-31f829c00337
mongo_1  | 2018-03-01T23:06:40.526+0000 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/db/diagnostic.data'
mongo_1  | 2018-03-01T23:06:40.527+0000 I NETWORK  [initandlisten] waiting for connections on port 27017
```

Logs for `mongo:3.6`:
```
$ docker-compose logs mongo
Attaching to mongodbexporter_mongo_1
mongo_1  | 2018-03-01T22:39:27.597+0000 I CONTROL  [initandlisten] MongoDB starting : pid=1 port=27017 dbpath=/data/db 64-bit host=2db857843c79
mongo_1  | 2018-03-01T22:39:27.599+0000 I CONTROL  [initandlisten] db version v3.6.2
mongo_1  | 2018-03-01T22:39:27.599+0000 I CONTROL  [initandlisten] git version: 489d177dbd0f0420a8ca04d39fd78d0a2c539420
mongo_1  | 2018-03-01T22:39:27.599+0000 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.1t  3 May 2016
mongo_1  | 2018-03-01T22:39:27.599+0000 I CONTROL  [initandlisten] allocator: tcmalloc
mongo_1  | 2018-03-01T22:39:27.599+0000 I CONTROL  [initandlisten] modules: none
mongo_1  | 2018-03-01T22:39:27.599+0000 I CONTROL  [initandlisten] build environment:
mongo_1  | 2018-03-01T22:39:27.599+0000 I CONTROL  [initandlisten]     distmod: debian81
mongo_1  | 2018-03-01T22:39:27.599+0000 I CONTROL  [initandlisten]     distarch: x86_64
mongo_1  | 2018-03-01T22:39:27.599+0000 I CONTROL  [initandlisten]     target_arch: x86_64
mongo_1  | 2018-03-01T22:39:27.599+0000 I CONTROL  [initandlisten] options: { net: { bindIpAll: true } }
mongo_1  | 2018-03-01T22:39:27.599+0000 I STORAGE  [initandlisten] 
mongo_1  | 2018-03-01T22:39:27.599+0000 I STORAGE  [initandlisten] ** WARNING: Using the XFS filesystem is strongly recommended with the WiredTiger storage engine
mongo_1  | 2018-03-01T22:39:27.599+0000 I STORAGE  [initandlisten] **          See http://dochub.mongodb.org/core/prodnotes-filesystem
mongo_1  | 2018-03-01T22:39:27.599+0000 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=1461M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
mongo_1  | 2018-03-01T22:39:27.667+0000 I CONTROL  [initandlisten] 
mongo_1  | 2018-03-01T22:39:27.667+0000 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
mongo_1  | 2018-03-01T22:39:27.667+0000 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
mongo_1  | 2018-03-01T22:39:27.667+0000 I CONTROL  [initandlisten] 
mongo_1  | 2018-03-01T22:39:27.679+0000 I STORAGE  [initandlisten] createCollection: admin.system.version with provided UUID: f02b1e9b-3d37-42a3-9dff-6accdfdeabb8
mongo_1  | 2018-03-01T22:39:27.722+0000 I COMMAND  [initandlisten] setting featureCompatibilityVersion to 3.6
mongo_1  | 2018-03-01T22:39:27.731+0000 I STORAGE  [initandlisten] createCollection: local.startup_log with generated UUID: 7a7170f0-5d2d-470c-956e-d0591275987d
mongo_1  | 2018-03-01T22:39:27.753+0000 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/db/diagnostic.data'
mongo_1  | 2018-03-01T22:39:27.755+0000 I NETWORK  [initandlisten] waiting for connections on port 27017
```

The only difference seems to be `options: { net: { bindIpAll: true } }`:
```
** WARNING: This server is bound to localhost.
**          Remote systems will be unable to connect to this server.
**          Start the server with --bind_ip <address> to specify which IP
**          addresses it should serve responses from, or with --bind_ip_all to
**          bind to all interfaces. If this behavior is desired, start the
**          server with --bind_ip 127.0.0.1 to disable this warning.
```

Reply:
https://github.com/docker-library/mongo/pull/226
> # MongoDB 3.6+ defaults to localhost-only binding

Should I create jira issue for this docker image? :)

Reply:
Please do.

Reply:
https://jira.percona.com/browse/BLD-991